### ISSUE TITLE
Cross Chain Attack

### ISSUE THREAD

**Issue by williamhsiao0403** [FinTechIntro/2024-Spring-Final#12]
## Summary
> Please give a short summary of this research topic.

Cross-Chain Bridges are technologies that allow the transfer of assets between different blockchains. This technology is a key component of blockchain interoperability, enabling users to freely exchange cryptocurrencies, tokens, data, and other resources across various blockchain platforms. However, there have been numerous significant security incidents on cross-chain bridges that have led to substantial financial losses. Please present your research report on some cross-chain bridge attack incidents.
## Description
> Please provide a detailed description to the topic.

Cross-chain bridge attacks exploit vulnerabilities in the technology connecting different blockchains to steal assets. These attacks often target flaws in smart contracts, including reentrancy attacks, logic errors, or data validation issues. For example, attackers may use reentrancy attacks to withdraw funds multiple times beyond the locked amount. They could also exploit logical flaws in smart contracts to lock or generate bottomless tokens, allowing these tokens to be reused on the target chain. Additionally, some attacks manipulate on-chain price mechanisms by providing false information, thus completing transactions at abnormal prices.

The success of these attacks often leads to significant financial losses, highlighting the urgent need for enhanced security measures in cross-chain operations. To prevent such attacks, developers of cross-chain bridges need to ensure that smart contracts undergo rigorous security audits and implement multiple verification and security mechanisms to protect user assets. 

Please select a cross-chain bridge attack incident as the subject of study and explain how the attack was implemented. If possible, further investigate how it can be prevented.
## Source
> Please provide related resource for the topics, including articles, research papers and more.
[rekt](https://rekt.news/zh/leaderboard/)
[BNB BRIDGE](https://rekt.news/zh/bnb-bridge-rekt/)
[NOMAD BRIDGE](https://rekt.news/nomad-rekt/)
## Others
> Please provide more information if there is any.

**Comment by hansliu11** [comment]
Hi TA! I would like to assign myself to this issue. Could you please add me as an assignee? Thanks

**Comment by LouisTsai-Csie** [comment]
@hansliu11 No problem, have fun with this research topic.

Reach out to @williamhsiao0403 if you encounter any issue.

**Comment by joel0115** [comment]
Hi TA, I would like to change my topic to this one due to the topic I selected before ([Phishing Scam on web3](https://github.com/FinTechIntro/2024-Spring-Final/issues/11)) are already selected by three people.
Sorry for the inconvenience.

**Comment by williamhsiao0403** [comment]
It doesn't  matter if someone  chose the topic or not, you can still do the
scam topic.  R u sure u wanna change ur topic?

joel0115 ***@***.***> 於 2024年5月26日 週日 上午12:36 寫道：

**Comment by joel0115** [comment]
Yes, I'm sure, the cross-chain attack topic is also interesting.
Thanks for helping!

williamhsiao0403 ***@***.***> 於 2024年5月26日 週日 02:37 寫道：

**Comment by l-akashi** [comment]
Hi TA! I want to assign myself to this issue. Could you please add me as an assignee? Thanks

**Comment by LouisTsai-Csie** [comment]
@l-akashi Done, would like to know which types of attack are you researching?

**Comment by l-akashi** [comment]
@LouisTsai-Csie I think that I will research the reentrancy attacks.

**Comment by LouisTsai-Csie** [comment]
@l-akashi Great, we anticipate you to reproduce these attacks through foundry and write detailed postmortems for the exploit.

**Comment by wangyc803** [comment]
Hello TA, I'm intrested in this topic and would like to research on it as my final project. Could you add me as an assignee? Thanks.
